app/assets/javascripts/discourse/app/templates/modal/dismiss-notification-confirmation.hbs

```diff
@@ -0,0 +1,8 @@
+{{#d-modal-body headerClass="hidden" class="dismiss-notification-confirmation"}}
+    {{i18n "notifications.dismiss_confirmation.body" count=count}}
+{{/d-modal-body}}
+
+<div class="modal-footer">
+  {{d-button icon="check" class="btn-primary" action=(action "confirm") label="notifications.dismiss_confirmation.confirm"}}
+  {{d-button action=(route-action "closeModal") label="notifications.dismiss_confirmation.cancel" class="cancel"}}
+</div>
```

Comment:
Some useful input would be if the modals have a standard with regards to titles, bodies, and buttons. The style guide doesn't seem to have an opinion.

Reply:
Oh sorry but you can use a `bootbox.confirm` for this, see:

https://github.com/discourse/discourse/blob/08e625c4469f297de21cbdfe78d133e0d550fd2a/app/assets/javascripts/admin/addon/controllers/admin-customize-themes-show.js#L392-L401

Reply:
Then you can delete the other modal stuff you have added.